File: Leetcode_Javascript/TwoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
	var number = [];

	for (let i = 0; i <= nums.length; i++) {
		var temp = nums[i];
		for (let j = i + 1; j <= nums.length - 1; j++) {
			if (temp + nums[j] == target) {
				number.push(i);
				number.push(j);
				return number;
				break;
			}
		}
	}
};
// Elias Kibret

File: Leetcode_Python/twosum.py
# Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to target.

# You may assume that each input would have exactly one solution, and you may not use the same element twice.

# You can return the answer in any order.



class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:       
        my_dict={}
        for index,value in enumerate(nums):
            comp=target-value
            if comp in my_dict:
                return [index,my_dict[comp]]
            my_dict[value]=index
        return []
                            #Elias Kibret

        